samples/bluetooth/mesh/sensor_client/README.rst

```diff
@@ -0,0 +1,168 @@
+
+  * `nRF Mesh mobile app for Android`_
+  * `nRF Mesh mobile app for iOS`_
+
+* :ref:`bluetooth_mesh_sensor_server` sample application programmed on a separate board and configured according to the Mesh Sensor Server sample's :ref:`bluetooth_mesh_sensor_server_testing` guide.
+
+User interface
+**************
+
+Buttons:
+   Can be used to input the OOB authentication value during provisioning.
+   All buttons have the same functionality during this procedure.
+Terminal:
+   All sensor values gathered from the Server is printed over UART. See :ref:`gs_testing`.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/mesh/sensor_client`
+
+.. include:: /includes/build_and_run.txt
+
+.. _bluetooth_mesh_sensor_client_testing:
+
+Testing
+=======
+
+.. important::
+   The Sensor Client sample cannot demonstrate any functionality on its own, and needs a device with the :ref:`bluetooth_mesh_sensor_server` sample running in the same Mesh network.
+   Before testing Mesh Sensor Client, go through the Mesh Sensor Server's :ref:`bluetooth_mesh_sensor_server_testing` guide with a different board.
+
+After programming the sample to your board, you can test the device by provisioning and configuring it for communication trough the Mesh models.
+This can be done using a smartphone with Nordic Semiconductor's nRF Mesh app installed.
+
+All sensor values gathered from the Server is printed over UART. See :ref:`gs_testing`.
+
+Provisioning the device
+-----------------------
+
+The provisioning assigns an address range to the device, and adds it to the Mesh network.
+Complete the following steps in the nRF Mesh app:
+
+1. Tap :guilabel:`Add node` to start scanning for unprovisioned Mesh devices.
+#. Select the :guilabel:`Mesh Sensor Client` device to connect to it.
+#. Tap :guilabel:`Identify` and then :guilabel:`Provision` to provision the device.
+#. When prompted, select an OOB method and follow the instructions in the app.
+
+Once the provisioning is complete, the app returns to the Network screen.
+
+Configuring models
+------------------
+
+Complete the following steps in the nRF Mesh app to configure models:
+
+1. On the Network screen, tap the :guilabel:`Mesh Sensor Client` node.
+   Basic information about the Mesh node and its configuration is displayed.
+#. In the Mesh node view, expand the element.
+   It contains the list of models in the first and only element of the node.
+#. Tap :guilabel:`Sensor Client` to see the model's configuration.
+#. Bind the model to application keys to make it open for communication:
+
+   1. Tap :guilabel:`BIND KEY` at the top of the screen.
+   #. Select :guilabel:`Application Key 1` from the list.
+
+#. Set the publishing parameters:
+
+   1. Tap :guilabel:`SET PUBLICATION`.
+   #. Tap :guilabel:`Publish Address`.
+   #. Choose Groups from the dropdown menu.
+   #. Choose an existing group or create a new. *IMPORTANT: The Sensor Server must subscribe to the same grop*.
```

Comment:
```suggestion
   #. Choose an existing group or create a new.
       *IMPORTANT: The sensor server must subscribe to the same group*.
```
Or you could consider adding a note under this list item instead of writing *IMPORTANT: The sensor server must subscribe to the same group*.